knowledge-base/capturing-git-traffic.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Capturing Git Traffic
+description: "Learn how to capture Git HTTPS traffic while using the Fiddler Everywhere web-debugging tool."
+type: how-to
+slug: how-to-capture-git-traffic
+publish: true
+res_type: kb
+---
+
+
+## Environment
+
+|   |   |
+|---|---|
+| Product version | Fiddler Everywhere 1.0.0 and above  |
+| 3rd-party tools | [GIT](https://git-scm.com/), [GitHub](https://github.com/) |
+| Supported OS | macOS, Linux, Windows |
+
+
+## Description
+
+I want to capture the traffic generated from the Git source control while making pull & push requests and cloning repositories. How to configure Git to go through the Fiddler Everywhere proxy?
+
+
+## Configuring Git HTTPS Access alongside Fiddler Everywhere Proxy
+
+1. Start your Git Bash terminal with administrative rights.
+
+1. Configure Git (for HTTPS-accessed repositories) to use the Fiddler Everywhere proxy.
+
+    ```
+    git config --global http.proxy http://127.0.0.1:8866 
+    ```
+
+1. Disable the SSL/TLS verification by setting `http.sslVerify` to `false`. Note that this will make your connection insecure.
```

Comment:
Is this required when the Fiddler's certificate is trusted? Isn't there a way to tell git to use the fiddler certificate?

Reply:
It is possible to explicitly set the Fiddler certificate through **http.sslCAInfo** or **http.sslCAPath**, but I removed that from the article for the sake of simplicity and because most developers are going with sslVerify instead of explicitly providing a certificate.

I will add this to the article as a possible long-term solution.